addons/laser/functions/fnc_laserPointTrack.sqf

```diff
@@ -0,0 +1,45 @@
+#include "script_component.hpp"
+/*
+ * Author: LorenLuke
+ * Toggles laser point tracking when a laser is on, for tracking coordinates;
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ *
+ * Return Value:
+ * Nothing
+ *
+ * Example:
+ * [vehicle player] call ace_laser_fnc_laserPointTrack
+ *
+ * Public: No
+ */
+ 
+
+params ["_vehicle"];
+
+GVAR(TrackerpfID) = [{
+    params ["_args", "_pfID"];
+    _args params ["_vehicle"];
+    if (!(hasPilotCamera _vehicle)) exitWith {
```

Comment:
```suggestion
    if (!(hasPilotCamera _vehicle) || {isNull (laserTarget _vehicle)}) exitWith {
```

Reply:
ehhhhhh... While that works within the scope of how things are currently, I want to keep the lines separate because of how the laser point system works for weird edge cases like the blackfish, which don't return a laserTarget.